utils/obsproc/Viirsaod2Ioda.h

```diff
@@ -132,12 +121,80 @@ namespace gdasapp {
                 }
                 obserror[i][j] = obserrorValue;
                 mask[i][j] = 1;
-                nobs += 1;
              }
           }
         }
       }
 
+      // superobing, not work yet...
```

Comment:
is this comment correct? or does it work now?

Reply:
This comment is removed.